
public class NestedTryBlock {

	public static void main(String[] args) {
			try{
			try{
			int a=10/0;
			}
			catch(ArithmeticException e){
			System.out.println(e);
			}
			try{
			int a[]=new int[5];
			a[6]=10;
			}
			catch(ArrayIndexOutOfBoundsException e){
			System.out.println("will handle arrayoutofboundexception");
			}
			System.out.println("Exception handled");
			}
}
}

